docs/migrations/web/MIGRATION-v3.md

```diff
@@ -6,9 +6,163 @@ Introducing version 3 of the _spirit-web_ package.
 
 ## Overview
 
+- [General Changes](#general-changes)
+  - [Themes](#themes)
+  - [Spacing values](#spacing-values)
+  - [Helpers: Typography `text` infix and Heading emphasis](#helpers-typography-text-infix-and-heading-emphasis)
+  - [Utilities: Removed and new classes](#utilities-removed-and-new-classes)
+  - [Migrating product components](#migrating-product-components)
 - [Component Changes](#component-changes)
   - [Button: renamed modifier `square`](#button-square-modifier-renamed-to-symmetrical)
 
+## General Changes
+
+## Themes
+
+Starting from version 3, the design system now supports multiple themes based on the
+theme tokens. As in Figma, the first theme is the default one.
```

Comment:
"As in Figma" - link to the Figma documentation where it is described will be handy.